src/accordion/accordion.component.ts

```diff
@@ -8,6 +15,12 @@ import { Component } from "@angular/core";
 		</ul>
 	`
 })
-export class Accordion {
+export class Accordion implements AfterContentInit {
+	@ContentChildren(AccordionItem) childs: QueryList<AccordionItem>;
 
+	@Input() skeleton = false;
+
+	ngAfterContentInit() {
+		this.childs.toArray().forEach(child => child.skeleton = this.skeleton);
```

Comment:
Same issue as breadcrumbs, we'll need something else to update the child state when the input changes. Also `children` not `childs`